common/swagger/swagger.yaml

```diff
@@ -64,6 +64,25 @@ paths:
           description: Successful operation
           schema:
             $ref: '#/definitions/ScoreRequest'
+  '/model/features':
+    get:
+      tags:
+        - enabled features
+      summary: List of features supported by the scorer
```

Comment:
Nitpick: i learned from Ondrej to finish each summary with `.` to avoid confusion (is the sentence complete)?

Reply:
Awesome - done.